apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -0,0 +1,123 @@
+import { Client } from "auth0";
+
+const USER_ID: string = "auth0|667c3d0c90b963e3671f411e";
+
+describe("Client API", () => {
+  const PORT: number = (Cypress.env("PORT") as number) || 8089;
+  let accessToken: string;
+  let m2mToken: string;
+  let newClientId: string;
+
+  const getTokens = () => {
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt-m2m")
+      .then((token: string) => {
+        if (token) {
+          m2mToken = token;
+        }
+      });
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt-client")
+      .then((token: string) => {
+        if (token) {
+          accessToken = token;
+        }
+      });
+  };
+
+  before(() => {
+    getTokens();
+    if (!accessToken) {
+      cy.loginM2MAuth0();
+      cy.wait(1500);
+    }
+    if (!m2mToken) {
+      cy.loginClientAuth0();
+      cy.wait(1500);
+    }
+    getTokens();
+  });
+
+  // "creates a client, fails if another client request is made, gets the newly created client, and deletes the client"
+  it("clears the client_id from user metadata, tries creating a client without access token, creates a client, fails if another client request is made, gets the newly created client, and deletes the client", () => {
+    cy.request({
```

Comment:
Does the clientId not get cleared from the user's metadata when you delete the client?

Reply:
This could be leaving orphaned clients out there if we're clearing a client that still exists from a user

Reply:
Yeah, I was thinking about this yesterday. I created a ticket for this. I'm not sure atm how to handle this. https://universalconnect.atlassian.net/browse/UCP-197

Reply:
I don't think deleting Clients will be a common occurrence, but I'll figure out a way to handle it when we do need to.

Reply:
I found out that you can search the user_metadata, so I'll be able to remove the client_id metadata for each user associated with a specific client.

Reply:
For now I would just remove it from the user's metadata who called the endpoint. When we implement organizations, then we can deal with removing it from other users who are attached to it.